Tests/Tests/DDFileLoggerTests.m

```diff
@@ -16,18 +16,87 @@
 #import <XCTest/XCTest.h>
 #import <CocoaLumberjack/CocoaLumberjack.h>
 
-@interface DDFileLoggerTests : XCTestCase
+#import "DDSampleFileManager.h"
+
+static const DDLogLevel ddLogLevel = DDLogLevelAll;
+
+@interface DDFileLoggerTests : XCTestCase {
+    DDFileLogger *logger;
+}
 
 @end
 
 @implementation DDFileLoggerTests
 
-- (void)testBuffer {
-    __auto_type logger = [DDFileLogger new];
+- (void)setUp {
+    [super setUp];
+    logger = [[DDFileLogger alloc] initWithLogFileManager:[[DDSampleFileManager alloc] initWithLogFileHeader:@"header"]];
+}
+
+- (void)tearDown {
+    [super tearDown];
+
+    for (NSString *logFilePaths in logger.logFileManager.unsortedLogFilePaths) {
+        NSError *error = nil;
+        XCTAssertTrue([[NSFileManager defaultManager] removeItemAtPath:logFilePaths error:&error]);
```

Comment:
Will XCTAsserts in `tearDown` fail the test?

Reply:
Just tested `XCTAssertTrue(false)` in there, and it works as expected. It is associated with the test that is being torn down. Quick/Nimble doesn't support this if I remember correctly.